third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -137,11 +137,6 @@ function juLog() {
   content="$content
     <testcase assertions=\"1\" name=\"$name\" time=\"$time\" classname=\"$class\">
     $failure
-    <system-out>
```

Comment:
No one else is using this now, so this change will not affect anyone else.
And stdout can be found in build log.